bin.js

```diff
@@ -3,22 +3,42 @@
 const gfm2html = require('./')
     , fs       = require('fs')
 
+var input = ''
+
 function usageAndExit () {
   console.error('Usage: gfm2html <markdown file> <output file>')
   process.exit(-1)
 }
 
-if (process.argv.length < 4)
-  return usageAndExit()
+function finish(input) {
+  gfm2html(input, function (err, data) {
+    if (err)
+      throw err
 
-if (!fs.existsSync(process.argv[2])) {
-  console.error('File "' + process.argv[2] + '" does not exist')
-  return usageAndExit()
+    if (process.argv[3] === '-') {
+      process.stdout.write(data);
```

Comment:
also remove braces from around this, don't use them when both the if and the else are single-line